common/config.py

```diff
@@ -99,6 +99,7 @@ def __init__(self):
         self.tls_ca_path = None
         self.public_port = 8000
         self.private_port = 8000
+        self.max_request_size = int(os.getenv("MAX_REQUEST_SIZE", "2"))  # MB
```

Comment:
I would suggest making it clear in the name in which measure is the size

Something like
`MAX_REQUEST_SIZE_MB` or `MAX_REQUEST_MB`